mitodl/open-discussions
# Added retire_users command and don't email or index inactive users

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1698
Fixes #2076 
Part of #2118

#### What's this PR do?
- Adds a mangement command to retire users
- Updates email notifications to not send to inactive users
- Disables user profile page if inactive
- Don't index inactive users

#### How should this be manually tested?
- Run `./manage.py retire_user [--email] [--username] [--id]` and verify the user is deactivated
- Verify the user doesn't receive daily or weekly email notifications
- Verify the user profile page results in a 404 page
- Run `./manage.py recreate_index`
